selfdrive/car/hyundai/interface.py

```diff
@@ -126,6 +126,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       tire_stiffness_factor = 0.385
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
+    elif candidate in CAR.IONIQ_EV_2020:
```

Comment:
Seems like all params are the same, just add it to the list with the other Ioniq models